### PR TITLE
Add License files for CC 1.0.0 & Remove unwanted CPP source files

### PR DESCRIPTION
**Pull request by renuka-fernando**
### Purpose
$subject + Remove unwanted CPP source files

### Issues

Task

### Automation tests
 - Unit tests added: N/A
 - Integration tests added: N/A

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [-] Validated respective github issues
- [-] Assigned milestone to the github issue(s)

### CODE REVIEW

**Comment by renuka-fernando** [comment]
TODO: Add enforcer license.txt file

**Review by AmaliMatharaarachchi — APPROVED** [review]

File: adapter/LICENSE.txt [hunk 1]
```diff
@@ -71,9 +83,8 @@ apache2        Apache License Version 2.0
 mit            MIT License
                http://www.opensource.org/licenses/mit-license.php
 bsd2           Berkeley License - 2
-               https://opensource.org/licenses/BSD-2-Clause
+               https://opensource.org/licenses/bsd2        
 bsd3           Berkeley License - 3
-               http://opensource.org/licenses/BSD-3-Clause
+               http://opensource.org/licenses/bsd3        
 isc	       Internet Systems Consortium		
 	       https://opensource.org/licenses/ISC
```
Comment (renuka-fernando) [inline adapter/LICENSE.txt:90]: This should be corrected

**Review by pubudu538 — APPROVED** [review]

**Review by praminda — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211215.32

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211215.43

**Pull request merged**